### PR TITLE
Volcanoplot redesign

### PR DESCRIPTION
**Pull request by shammamah**
## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: assets/volcanoplot-style.css [hunk 1]
```diff
@@ -1,106 +1,142 @@
...
+    margin: 20px;
 }
 
-.vp-horizontal-style {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
+#vp-control-tabs .tab {
+    background-color: #fff;
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
    background-color: #FFFFFF;
```
for sake of notation uniformity

File: assets/volcanoplot-style.css [hunk 2]
```diff
@@ -1,106 +1,142 @@
...
+    padding: 15px;
 }
 
-.vp-input{
-    font-size: 1.5em;
-    height: 30px;
-    width: 80px;
-    margin-top: 20px;
+#vp-control-tabs .tab--selected {
+    border-top: solid 2px #19d3f3;
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
    border-top: solid 2px #19D3F3;
```

File: assets/volcanoplot-style.css [hunk 3]
```diff
@@ -1,106 +1,142 @@
...
+    border-bottom: none;
+    color: #19d3f3;
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
    color: #19D3F3;
```

File: assets/volcanoplot-style.css [hunk 4]
```diff
@@ -1,106 +1,142 @@
...
-    border-radius: 0px !important;
-
+#vp-page-content .Select input {
+    padding: 0px !important;
 }
-
-#vp-color-label {
-    margin-right: 5%;
-    margin-left: 5%;
+#vp-graph{
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
#vp-graph {
```

File: assets/volcanoplot-style.css [hunk 5]
```diff
@@ -1,106 +1,142 @@
...
+    font-size: 12pt;
+    display: inline-block;
+    vertical-align: top;
+    margin-top: 15px;
+    margin-bottom: 10px;
+    width: 100px;
+}
+
+#vp-page-content .rc-slider-track {
+    background-color: #19d3f3;
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
    background-color: #19D3F3;
```

File: assets/volcanoplot-style.css [hunk 6]
```diff
@@ -1,106 +1,142 @@
...
+}
+#vp-page-content .rc-slider-handle, #vp-page-content .rc-slider-dot-active {
+    border-color: #19d3f3;
```
Comment (mkcor) [inline assets/volcanoplot-style.css]:
```suggestion
    border-color: #19D3F3;
```

File: tests/dashbio_demos/app_volcano_plot.py [hunk 7]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+                        html.H4('What is Volcano Plot?'),
+                        html.P(
+                            'You can use volcano plot to interactively '
+                            'identify clinically meaningful markers in '
+                            'genomic experiments, i.e., markers that are '
+                            'statistically significant and have an effect '
+                            'size greater than some threshold. '
+                            'Specifically, volcano plots depict the negative '
+                            'log-base-10 p-values plotted against their '
+                            'effect size, odds ratio, or log fold-change. '
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]:
```suggestion
                            'effect size.'
```
I know this is not the focus of the PR, but I would like to take this opportunity to fix the description: Volcano plots may conventionally be used to plot ... against one of "effect size, odds ratio, or log fold-change" but, in the current version of this demo app, I can't find any way to change the x-axis to represent odds ratio or log fold-change, so I find it deceiving to mention them in this context.

File: tests/dashbio_demos/app_volcano_plot.py [hunk 8]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+        ),
+        html.Div(id='vp-control-tabs', children=[
+            dcc.Tabs(id='vp-tabs', value='what-is', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='vp-tab', children=[
+                        html.H4('What is Volcano Plot?'),
+                        html.P(
+                            'You can use volcano plot to interactively '
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]:
```suggestion
                            'You can use Volcano Plot to interactively '
```
(or, alternatively, "a volcano plot")
I would also drop "you can"

File: tests/dashbio_demos/app_volcano_plot.py [hunk 9]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+                            'log-base-10 p-values plotted against their '
+                            'effect size, odds ratio, or log fold-change. '
+                        ),
+                        html.P(
+                            'In the "Data" tab, you can select a dataset '
+                            'to view on the plot. In the "View" tab, you '
+                            'can control the color of the highlighted '
+                            'points, as well as the threshold lines that '
+                            'define which values are significant. You can '
+                            'also access hover data and click data from '
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]:
```suggestion
                            'also access metadata from hovering and '
```

File: tests/dashbio_demos/app_volcano_plot.py [hunk 10]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+                            'your interactions with the graph. '
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]:
```suggestion
                            'clicking on the graph.'
```

File: tests/dashbio_demos/app_volcano_plot.py [hunk 11]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
-                                        id='vp-upper-left',
-                                        size=20,
-                                    ),
-                                    html.Div(
-                                        className='vp-test-util-div',
-                                        id='vp-upper-left-val'
-                                    )
-                                ],
+                                className='vp-option-name',
+                                children='Effect sizes'
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]:
```suggestion
                                children='Effect size bounds'
```

File: tests/dashbio_demos/app_volcano_plot.py [hunk 12]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+            html.Div(className='vp-option-name', children='Hover data')
+        ]
+        hover_data = 'Hover over a data point to see it here.'
+
+        if hover is not None:
+            hovered_point = hover['points'][0]
+            hover_data = [
+                'x: {}'.format(hovered_point['x']),
+                html.Br(),
+                'y: {}'.format(hovered_point['y']),
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]: Could we keep fewer decimals for the y coordinate?
y_decimals

File: tests/dashbio_demos/app_volcano_plot.py [hunk 13]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+            html.Div(className='vp-option-name', children='Click data')
+        ]
+        click_data = 'Click on a data point to see it here.'
+
+        if click is not None:
+            clicked_point = click['points'][0]
+            click_data = [
+                'x: {}'.format(clicked_point['x']),
+                html.Br(),
+                'y: {}'.format(clicked_point['y']),
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]: same question as above

File: tests/dashbio_demos/app_volcano_plot.py [hunk 14]
```diff
@@ -65,213 +65,212 @@ def header_colors():
...
+        hover_data = 'Hover over a data point to see it here.'
+
+        if hover is not None:
+            hovered_point = hover['points'][0]
+            hover_data = [
+                'x: {}'.format(hovered_point['x']),
+                html.Br(),
+                'y: {}'.format(hovered_point['y']),
+                html.Br(),
+                hovered_point['text'].strip('<br>').replace(
```
Comment (mkcor) [inline tests/dashbio_demos/app_volcano_plot.py]: Is this space-padded hyphen standard gene notation?

File: tests/test_volcano_plot.py [hunk 15]
```diff
@@ -1,96 +1,17 @@
...
     template_test_python_component_prop,
     PROP_TYPES
 )
 
 APP_NAME = os.path.basename(__file__).replace('test_', '').replace('.py', '').replace('_', '-')
 
 
-# Demo app tests
-@init_demo_app(APP_NAME)
-def test_click_app_link_from_gallery(dash_threaded):
```
Comment (mkcor) [inline tests/test_volcano_plot.py]: Actually, we currently have this test for other demo apps. Are you sure you want to take it out?

File: tests/dashbio_demos/app_volcano_plot.py (hunk 14 above)
Comment (shammamah) [reply tests/dashbio_demos/app_volcano_plot.py]: Not sure, but they were originally separated by a line break, which I don't think is "proper" notation -- this is just a different separator. I can replace the hyphen with putting parentheses around it (done in 7e90265)

File: tests/dashbio_demos/app_volcano_plot.py (hunk 13 above)
Comment (shammamah) [reply tests/dashbio_demos/app_volcano_plot.py]: Fixed in 7e90265

File: tests/dashbio_demos/app_volcano_plot.py (hunk 12 above)
Comment (shammamah) [reply tests/dashbio_demos/app_volcano_plot.py]: Fixed in 7e90265

File: tests/test_volcano_plot.py (hunk 15 above)
Comment (shammamah) [reply tests/test_volcano_plot.py]: Whoops! Good catch :)

File: tests/dashbio_demos/app_volcano_plot.py (hunk 14 above)
Comment (shammamah) [reply tests/dashbio_demos/app_volcano_plot.py]: Yeah for sure! It's good to catch things that might look unclear. I don't have a lot of experience with bioinformatics, so I'm just as unsure as you are about these things! :P

Comment (mkcor) [reply tests/dashbio_demos/app_volcano_plot.py]: Ok, anyway, I personally like how it renders now, so I'll resolve... Thanks!

**Pull request merged**